frontend/src/components/Tournament.js

```diff
@@ -24,6 +25,51 @@ import {
 } from "../skeletons/Standings";
 import Breadcrumbs from "./Breadcrumbs";
 
+const TeamSeedingChange = props => {
+  const filteredByTeamID = Object.entries(
+    props.tournament.initial_seeding
+  ).filter(([_, teamID]) => teamID == props.teamID); // eslint-disable-line no-unused-vars
+
+  const [initialSeed, _] = filteredByTeamID[0]; // eslint-disable-line no-unused-vars
```

Comment:
Any way we can remove the eslint disable comments? I see there are no unused vars anyway

Reply:
underscores are also coming as unused vars in eslint :disappointed:

Reply:
ahh cool